code/_globalvars/lists/names.dm

```diff
@@ -11,9 +11,10 @@ var/list/lizard_names_male = file2list("config/names/lizard_male.txt")
 var/list/lizard_names_female = file2list("config/names/lizard_female.txt")
 var/list/clown_names = file2list("config/names/clown.txt")
 var/list/mime_names = file2list("config/names/mime.txt")
+var/list/golem_names = file2list("config/names/golem.txt")

```

Comment:
Ah, but my other PR is PROBABLY only going to be for april fools. Unless people want to keep it for badminry. And if either one is accepted to master, it's not a very complicated merge.